Description: Install pkgconfig as data
 Upstream installs pkgconfig files as executable which lintian correctly complains
 about:
 W: libsnmp-dev: executable-not-elf-or-script usr/lib/x86_64-linux-gnu/pkgconfig/netsnmp-agent.pc
 W: libsnmp-dev: executable-not-elf-or-script usr/lib/x86_64-linux-gnu/pkgconfig/netsnmp.pc
 INSTALL_DATA installs files as 0644
Author: Craig Small <csmall@debian.org>
Forwarded: https://github.com/net-snmp/net-snmp/issues/168
Origin: Debian
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2020-08-22
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -298,7 +298,7 @@
 		for i in $(INSTALL_PKGCONFIG); do			\
 			echo "installing $$i in $(INSTALL_PREFIX)$(libdir)/pkgconfig"; \
 		done;							\
-		$(INSTALL) $(INSTALL_PKGCONFIG) $(INSTALL_PREFIX)$(libdir)/pkgconfig; \
+		$(INSTALL_DATA) $(INSTALL_PKGCONFIG) $(INSTALL_PREFIX)$(libdir)/pkgconfig; \
 	fi
 
 uninstall_pkgconfig:
